backend/core-api/src/modules/broadcast/utils.ts

```
@@ -0,0 +1,368 @@
+      .catch((e) => {
+        debugError(e.message);
+
+        if (e.message.includes('already exists')) {
+          return;
+        }
+
+        return reject(e.message);
+      });
+
+    return resolve(true);
+  });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)